js/bit2c.js

```diff
@@ -183,22 +183,19 @@ module.exports = class bit2c extends Exchange {
     parseTicker (ticker, market = undefined) {
         const symbol = this.safeSymbol (undefined, market);
         const timestamp = this.milliseconds ();
-        const averagePrice = this.safeNumber (ticker, 'av');
-        const baseVolume = this.safeNumber (ticker, 'a');
-        let quoteVolume = undefined;
-        if (baseVolume !== undefined && averagePrice !== undefined) {
-            quoteVolume = baseVolume * averagePrice;
-        }
-        const last = this.safeNumber (ticker, 'll');
+        const averagePrice = this.safeString (ticker, 'av');
+        const baseVolume = this.safeString (ticker, 'a');
+        const quoteVolume = undefined;
```

Comment:
We don't really need this variable, if it's `undefined`, let's move this value down to the return statement.